docs/AboutUs.md

```diff
@@ -9,15 +9,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Aryan Sarswat
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/aryansarswat.png" width="200px">
```

Comment:
I think the photo may not comply with the instructions - your face is rather small relative to the rest of the photo.
"The purpose of the profile photo is for the reader to identify you. Therefore, choose a recent individual photo showing your face clearly (i.e., not too small) -- somewhat similar to a passport photo."

Reply:
Changed!